lib/img/aws.js

```diff
@@ -45,6 +45,29 @@ AwsDriver.prototype = {
         }
       });
     });
+  },
+
+  delete: function awsDelete(key) {
```

Comment:
Accidentally passing through malicious keys with e.g. `..` or `/` or whatever in them; maybe less of a concern with the s3 backend though